customizations/README.md

```diff
@@ -536,6 +542,8 @@ In order to use Azure Storage buckets (aka containers) you need to:
 
 3. Create the storage buckets. If you set the `Public Access Mode` to `private` in the bucket properties, make sure you set `appConfigValues.workspaceThumbnailsPublic` to `false`.
```

Comment:
Now that you have added the `warning`, in step number 3 i think it's enought to leave `create the storage buckets`, other wise it is redundant.